lib/chef/provider/user/dscl.rb

```diff
@@ -195,9 +195,10 @@ def dscl_create_user
 
         #
         # Saves the specified Chef user `comment` into RealName attribute
-        # of Mac user.
+        # of Mac user. If `comment` is not specified, it takes `username` value.
         #
         def dscl_create_comment
+          @new_resource.comment(@new_resource.username) if @new_resource.comment.nil?
```

Comment:
@coderanger I've tried to replace this line by that:
```
return if @new_resource.comment.nil?
```
but got the same result as on the screenshot: "RealName" is empty and is not displayed in GUI